src/Agent.Plugins/BuildArtifact/BuildArtifactPluginV1.cs

```diff
@@ -64,6 +64,13 @@ protected static class TaskProperties
         }
     }
 
+    // Create a new version as we changed functionality for FileID https://portal.microsofticm.com/imp/v3/incidents/details/462028809/home
+    // For backcompat with older agents, tasks will use the node (v0) task code using AgentPluginWithFallback
+    public class DownloadBuildArtifactTaskV1_0_1 : DownloadBuildArtifactTaskV1_0_0
```

Comment:
Logging a comment to Hold on this change, as we consider another option (server side re-direct instead of fallback in task/agent)